platform/admin/ui/src/main/webapp/js/views/configuration/Service.view.js

```diff
@@ -62,6 +62,9 @@ define([
                     configuration.close();
                 });
             }
+        },
+        serializeData: function () {
```

Comment:
Do we need to serialize the `id` here? Can't we just get it back from the model when needed?